### PR TITLE
Provide code to replace Scene3D plugin and update Migration.md

### LINKED ISSUE #484: Use Migration.md instructions for replacing GzScene3D in Garden lead to a crash
## Environment
* OS Version: Focal
* Binaries in prerelease for Garden

## Description

While working on https://github.com/gazebosim/garden-tutorial-party/issues/2015 I was updating the GzScene3D plugin on https://github.com/gazebosim/docs/blob/master/garden/tutorials/sdf_worlds/world_demo.sdf#L23-L36. I checked the instructions in `Migration.md` and replaced by with https://github.com/gazebosim/docs/commit/d249a113ef0d2b238eb8ba609d0542044701409e, mostly TransportSceneManager and MinimalScene.

If I used that modified file, the result is:
image
(nothing displayed).

If I try to interact with the Scene, the simulator crashes:
```
terminate called after throwing an instance of 'Ogre::InvalidParametersException'
  what():  OGRE EXCEPTION(2:InvalidParametersException): Texture 'scene::RenderTexture(65526)0': Invalid settings! in TextureGpu::checkValidSettings at /home/jenkins/workspace/ogre-2.3-debbuilder/repo/OgreMain/src/OgreTextureGpu.cpp (line 451)
Stack trace (most recent call last) in thread 1108751:
#9    Object "[0xffffffffffffffff]", at 0xffffffffffffffff, in 
#8    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7f6411e37132, in clone
#7    Object "/lib/x86_64-linux-gnu/libpthread.so.0", at 0x7f6411cfd608, in 
#6    Object "/usr/lib/x86_64-linux-gnu/libQt5Core.so.5", at 0x7f640b98296d, in 
#5    Object "/usr/lib/x86_64-linux-gnu/libQt5Core.so.5", at 0x7f640b980e8a, in qTerminate()
#4    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6", at 0x7f640d8dd3f6, in std::terminate()
#3    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6", at 0x7f640d8dd38b, in 
#2    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6", at 0x7f640d8d1910, in 
#1    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7f6411d3a858, in abort
#0    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7f6411d5b00b, in gsignal
```

Note that the instructions in Migration.md are different from the behavior applied automatically if the GzScene3D is still in the sdf file:
```
[GUI] [Wrn] [Gui.cc:460] The [GzScene3D] GUI plugin has been removed since Garden. Loading the following plugins instead:
* GzSceneManager
* InteractiveViewControl
* CameraTracking
* MarkerManager
* SelectEntities
* EntityContextMenuPlugin
* Spawn
* VisualizationCapabilities
```

Not sure if I'm doing something wrong.

## Steps to reproduce

1. wget https://raw.githubusercontent.com/gazebosim/docs/master/garden/tutorials/sdf_worlds/world_demo.sdf
2. gz sim --verbose 4 world_demo.sdf 

## Output
```
gz sim --verbose 2 garden/tutorials/sdf_worlds/world_demo.sdf
[Wrn] [LocalCache.cc:105] Server directory does not exist [/home/jrivero/.gz/fuel/fuel.gazebosim.org]
[GUI] [Wrn] [Application.cc:845] [QT] file::/Gazebo/GazeboDrawer.qml:147:3: QML Dialog: Binding loop detected for property "implicitHeight"
[GUI] [Wrn] [Application.cc:845] [QT] file::/Gazebo/GazeboDrawer.qml:147:3: QML Dialog: Binding loop detected for property "implicitHeight"
[Wrn] [LocalCache.cc:105] Server directory does not exist [/home/jrivero/.gz/fuel/fuel.gazebosim.org]
[Wrn] [Component.hh:144] Trying to serialize component with data type [N3sdf3v135WorldE], which doesn't have `operator<<`. Component will not be serialized.
[GUI] [Wrn] [Application.cc:845] [QT] file::/EntityTree/EntityTree.qml:148:7: QML ToolButton: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
[GUI] [Wrn] [Component.hh:189] Trying to deserialize component with data type [N3sdf3v135WorldE], which doesn't have `operator>>`. Component will not be deserialized.
[GUI] [Wrn] [Application.cc:845] [QT] file::/Gazebo/GazeboDrawer.qml:147:3: QML Dialog: Binding loop detected for property "implicitHeight"
libEGL warning: DRI2: failed to create dri screen
libEGL warning: DRI2: failed to create dri screen
[GUI] [Wrn] [Application.cc:845] [QT] file::/Gazebo/GazeboDrawer.qml:147:3: QML Dialog: Binding loop detected for property "implicitHeight"
```

### PR DESCRIPTION
**Pull request by j-rivero**
# 🦟 Bug fix

Fixes #484 

## Summary

Include code to replace the removed Scene3D plugin so users can just copy/paste. I've updated the Migration.md, there was an old entry from 5.x to 6.x that I'm not sure if can be overwritten.

I would also love to update the console message to point to the code snippet but I was unable to find where the error comes from, gz-sim I assume.

```
[GUI] [Wrn] [Gui.cc:460] The [GzScene3D] GUI plugin has been removed since Garden. Loading the following plugins instead:
* GzSceneManager
* InteractiveViewControl
* CameraTracking
* MarkerManager
* SelectEntities
* EntityContextMenuPlugin
* Spawn
* VisualizationCapabilities
```

## Checklist
- [x] Signed all commits for DCO
- [x] Updated migration guide (as needed)
**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by ahcorde — CHANGES_REQUESTED** [review]
Add a link in the [Migration.md](https://github.com/gazebosim/gz-gui/blob/ign-gui6/Migration.md). Otherwise It's difficult to find this file.

**Comment by j-rivero** [comment]
> Add a link in the [Migration.md](https://github.com/gazebosim/gz-gui/blob/ign-gui6/Migration.md). Otherwise It's difficult to find this file.

Ouch, I forgot it [8fe6c91](https://github.com/gazebosim/gz-gui/pull/485/commits/8fe6c919b8f1efa4d18e47efc359e02db6d0939b)

File: examples/migrations/scene3d_deprecation.sdf [hunk 1]
```diff
@@ -0,0 +1,100 @@
+  <!-- 3D scene -->
```
Comment (jennuine) [inline examples/migrations/scene3d_deprecation.sdf:1]: Several of these plugins are `gz-sim` specific. I suggest removing this file and in the `Migration.md` replace the provided link with this instead: https://github.com/gazebosim/gz-sim/blob/ff1c82b41e548dfdc8076374f9500db2df2c35a1/examples/worlds/minimal_scene.sdf#L29-L128

**Review by ahcorde — APPROVED** [review]
LGTM, but CI is red :red_circle:

File: Migration.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ release will remove the deprecated code.
 
 * QML `import ignition.gui 1.0` is deprecated, use `import gz.gui 1.0` instead.
 
+* The `Scene3D` plugin as been removed. A possible replacement of SDF code can
+  be found in:
+  https://github.com/gazebosim/gz-sim/tree/main/examples/migrations/scene3d_deprecation.sdf
```
Comment (jennuine) [inline Migration.md:33]:
```suggestion
* The `Scene3D` plugin from [gz-sim](https://github.com/gazebosim/gz-sim) has been removed and replaced with `gz-gui`'s `MinimalScene` plugin. A possible replacement of SDF code can
  be found in:
  https://github.com/gazebosim/gz-sim/blob/ff1c82b41e548dfdc8076374f9500db2df2c35a1/examples/worlds/minimal_scene.sdf#L29-L128
```

**Review by jennuine — CHANGES_REQUESTED** [review]

**Review by ahcorde — CHANGES_REQUESTED** [review]
I think you should target `gz-sim7`

File: examples/migrations/scene3d_deprecation.sdf (hunk 1 above)
Comment (j-rivero) [reply examples/migrations/scene3d_deprecation.sdf]: sounds good, thanks [2e2807c](https://github.com/gazebosim/gz-gui/pull/485/commits/2e2807c0ad92a49c080a9e68408044948e7df810)

**Comment by j-rivero** [comment]
Follow in https://github.com/gazebosim/gz-sim/pull/1698

**Pull request closed without merging**